WebExample/tests/textManipulation.spec.ts

```diff
@@ -0,0 +1,139 @@
+});
+
+test.describe('paste content', () => {
+  test.skip(({browserName}) => !!process.env.CI && browserName === 'webkit', 'Excluded from webkit CI tests');
+
+  test('paste', async ({page}) => {
+    const PASTE_TEXT = 'bold';
+    const boldStyleDefinition = TEST_CONST.MARKDOWN_STYLE_DEFINITIONS.bold;
+
+    const inputLocator = await setupInput(page, 'clear');
+
+    const wrappedText = boldStyleDefinition.wrapContent(PASTE_TEXT);
+    await pasteContent({text: wrappedText, page, inputLocator});
+
+    const elementHandle = await inputLocator.locator('span', {hasText: PASTE_TEXT}).last();
+    let elementStyle;
+    if (elementHandle) {
+      await elementHandle.waitFor({state: 'attached'});
+
+      elementStyle = await elementHandle.getAttribute('style');
+    }
+    expect(elementStyle).toEqual(boldStyleDefinition.style);
+  });
+
+  test('paste replace', async ({page}) => {
+    const inputLocator = await setupInput(page, 'reset');
+
+    await inputLocator.focus();
+    await inputLocator.press(`${OPERATION_MODIFIER}+a`);
+
+    const newText = '*bold*';
+    await pasteContent({text: newText, page, inputLocator});
+
+    expect(await inputLocator.innerText()).toBe(newText);
+  });
+
+  test('paste undo', async ({page, browserName}) => {
+    test.skip(!!process.env.CI && browserName === 'firefox', 'Excluded from firefox CI tests');
+
+    const PASTE_TEXT_FIRST = '*bold*';
+    const PASTE_TEXT_SECOND = '@here';
+
+    const inputLocator = await setupInput(page, 'clear');
+
+    await page.evaluate(async (pasteText) => navigator.clipboard.writeText(pasteText), PASTE_TEXT_FIRST);
+
+    await inputLocator.press(`${OPERATION_MODIFIER}+v`);
+    await page.waitForTimeout(TEST_CONST.INPUT_HISTORY_DEBOUNCE_TIME_MS);
+    await page.evaluate(async (pasteText) => navigator.clipboard.writeText(pasteText), PASTE_TEXT_SECOND);
+    await inputLocator.press(`${OPERATION_MODIFIER}+v`);
+    await page.waitForTimeout(TEST_CONST.INPUT_HISTORY_DEBOUNCE_TIME_MS);
+
+    await inputLocator.press(`${OPERATION_MODIFIER}+z`);
+
+    expect(await inputLocator.innerText()).toBe(PASTE_TEXT_FIRST);
+  });
+
+  test('paste redo', async ({page}) => {
+    const PASTE_TEXT_FIRST = '*bold*';
+    const PASTE_TEXT_SECOND = '@here';
+
+    const inputLocator = await setupInput(page, 'clear');
+
+    await page.evaluate(async (pasteText) => navigator.clipboard.writeText(pasteText), PASTE_TEXT_FIRST);
+    await inputLocator.press(`${OPERATION_MODIFIER}+v`);
+    await page.waitForTimeout(TEST_CONST.INPUT_HISTORY_DEBOUNCE_TIME_MS);
+    await page.evaluate(async (pasteText) => navigator.clipboard.writeText(pasteText), PASTE_TEXT_SECOND);
+    await page.waitForTimeout(TEST_CONST.INPUT_HISTORY_DEBOUNCE_TIME_MS);
+    await inputLocator.press(`${OPERATION_MODIFIER}+v`);
+    await page.waitForTimeout(TEST_CONST.INPUT_HISTORY_DEBOUNCE_TIME_MS);
+
+    await inputLocator.press(`${OPERATION_MODIFIER}+z`);
```

Comment:
Instead of using `${OPERATION_MODIFIER}+x` everywhere, can we move it to an utility function `await pressCmd('x')`?